core/templates/pages/collection-editor-page/navbar/collection-editor-navbar.directive.ts

```diff
@@ -205,11 +205,6 @@ angular.module('oppia').directive('collectionEditorNavbar', [
             RouterService.navigateToHistoryTab();
           };
           ctrl.$onInit = function() {
-            ctrl.directiveSubscriptions.add(
-              CollectionEditorStateService.onCollectionInitialized.subscribe(
-                () => _validateCollection()
-              )
-            );
             ctrl.directiveSubscriptions.add(
```

Comment:
@kevintab95  this function call might be checking the collection after intialisation, I'm not sure though.

Reply:
I see. Can you please reply to my https://github.com/oppia/oppia/pull/10769#discussion_r504971836? Also, have you checked if the validation method is called when you go to the collection editor for an existing collection?

Reply:
I haven't checked it yet, but let me see. will get back to you once I found something.